examples/polynomial/presampled_evaluation/ensemble.yml

```diff
@@ -1,4 +1,4 @@
-size: 1000
+size: 100
```

Comment:
If it helps on test speed this can be reduced to `10` or `20` :)

Reply:
It probably will, thanks :smile:

Reply:
Reducing from 100 to 10 saved another 10-15s